packages/docs/docs/02-Guides/02-Reading/03-Handling Falsy.mdx

```diff
@@ -0,0 +1,33 @@
+# Handling Falsy types
+
+`useDapp` supports handling `False`
+
+```tsx
+export type Falsy = false | 0 | '' | null | undefined
+```
+
+If you pass one of these types to call, it will return undefined without making blockchain query.
```

Comment:
```suggestion
If you pass one of these types to a call, it will return `undefined` without making blockchain query. This allows us to conditionally make a call if a parameter is not known ahead of time, without breaking the [Rules of Hooks](https://reactjs.org/docs/hooks-rules.html).
```